backend/email/emailSender.ts

```diff
@@ -0,0 +1,29 @@
+// Source: https://mailtrap.io/blog/nodemailer-gmail/
+// If not working, unlock captcha and try again: https://accounts.google.com/b/0/displayunlockcaptcha
+
+import nodemailer from "nodemailer";
+import { SendMailOptions, Transporter } from "nodemailer";
+
+const transporter: Transporter = nodemailer.createTransport({
+  service: "gmail",
+  auth: {
+    user: "UKG.No.Reply@gmail.com",
+    pass: "kronos1234",
```

Comment:
This password is stored in plaintext on a public git repository. Potential solution: read an environment variable which is specified in a .env which setup instructions specify should be created. Then change the actual password

Reply:
I know I should use .env to avoid this, but... TBH I feel a little lazy to do that... Could we just have this part as it is..? lol